docs-src/go/how-to-use-worker-versioning-in-go.md

```diff
@@ -115,3 +115,22 @@ var yourActivityResult YourActivityResultType
 err := workflow.ExecuteActivity(ctx, YourActivityDefinition, yourActivityParam).Get(ctx, &yourActivityResult)
 // ...
 ```
+
+### Specifying Versioning for Continue-As-New
+
+When using the Continue-As-New, specify the versioning intent:
+
+1. Use the `WithWorkflowVersioningIntent` context modifier:
+   ```go
+   ctx = WithWorkflowVersioningIntent(ctx, temporal.VersioningIntentLatest)
+   err := workflow.NewContinueAsNewError(ctx, "WorkflowName")
+   ```
+
+2. The function `WithWorkflowVersioningIntent` sets the `VersioningIntent` prior to constructing a `ContinueAsNewError` with `NewContinueAsNewError`:
+   ```go
+   func WithWorkflowVersioningIntent(ctx Context, intent temporal.VersioningIntent) Context {
+       return internal.WithWorkflowVersioningIntent(ctx, intent)
+   }
+   ```
```

Comment:
there's no need to show how it's defined.. that doesn't add anything useful. maybe just use the text here on top of the first code block since it's a little more informative
```suggestion
```